### PR TITLE
Fb - install pico on webhosting

### PR DESCRIPTION
**Pull request by fabienOVH**
- Update the guide "how to install manually a CMS"
- New guide "How to install Pico"

### CODE REVIEW

File: pages/web_cloud/web_hosting/cms_manual_installation/guide.fr-fr.md [hunk 1]
```diff
@@ -75,6 +76,12 @@ Retrouvez plus d'informations sur notre page relative au [module PrestaShop](htt
 
 - Site officiel de [PrestaShop](https://www.prestashop.com/){.external}
 
+#### Pico
+
+**Pico** est un CMS léger basé sur PHP, idéal pour créer des sites web ou des blogs. Dépourvu de base de données, il utilise des fichiers Markdown pour gérer le contenu. Simple mais efficace, il supporte des extensions pour personnaliser votre site web.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/cms_manual_installation/guide.fr-fr.md]:
```suggestion
**Pico** est un CMS léger basé sur PHP, idéal pour créer des sites web ou des blogs. Dépourvu de base de données, il utilise des fichiers Markdown pour gérer le contenu. Il supporte des extensions pour personnaliser votre site web.
```

File: pages/web_cloud/web_hosting/cms_manual_installation/guide.fr-fr.md [hunk 2]
```diff
@@ -123,9 +130,19 @@ Si vous ne parvenez pas à effectuer ces vérifications, contactez l'hébergeur
 
 Si vous n'utilisez pas déjà un client FTP, vous pouvez utiliser Filezilla. Retrouvez le lien de téléchargement gratuit ainsi qu'un tutoriel sur son utilisation dans notre documentation sur l'[utilisation de FileZilla avec votre offre d'hébergement OVHcloud](/pages/web_cloud/web_hosting/ftp_filezilla_user_guide).
 
+> [!primary]
+>
+> Il existe d'autres clients FTP que vous pouvez utiliser, comme par exemple Cyberduck. Retrouvez le lien de téléchargement gratuit ainsi qu'un tutoriel sur son utilisation dans notre documentation sur l'[utilisation de Cyberduck avec votre offre d'hébergement OVHcloud](/pages/web_cloud/web_hosting/ftp_cyberduck_user_guide_on_mac).
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/cms_manual_installation/guide.fr-fr.md]:
```suggestion
> Il existe d'autres clients FTP que vous pouvez utiliser comme, par exemple, Cyberduck. Retrouvez plus d'informations dans notre documentation relative à l'[utilisation de Cyberduck avec votre offre d'hébergement OVHcloud](/pages/web_cloud/web_hosting/ftp_cyberduck_user_guide_on_mac).
```

File: pages/web_cloud/web_hosting/cms_manual_installation/guide.fr-fr.md [hunk 3]
```diff
@@ -185,7 +204,7 @@ Suivez les instructions indiquées par l'éditeur de votre CMS jusqu'à ce que l
 
 >[!primary]
 >
-> Pour plus de facilité, remplacez, dans cette étape, le nom du dossier « **CMS** » par le nom du CMS que vous avez choisi pour plus de facilité. (**WordPress**, **Joomla!**, **Drupal**, **PrestaShop**).
+> Pour plus de facilité, remplacez, dans cette étape, le nom du dossier « **CMS** » par le nom du CMS que vous avez choisi pour plus de facilité (**WordPress**, **Joomla!**, **Drupal**, **PrestaShop**, etc.).
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/cms_manual_installation/guide.fr-fr.md]:
```suggestion
> Pour plus de facilité, remplacez, dans cette étape, le nom du dossier « **CMS** » par le nom du CMS que vous avez choisi (**WordPress**, **Joomla!**, **Drupal**, **PrestaShop**, etc.).
```

File: pages/web_cloud/web_hosting/install-pico-cms/guide.fr-fr.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
+---
+title: "Tutoriel - Installer manuellement Pico"
+excerpt: "Découvrez comment installer le CMS Pico sur un hérgement web OVHcloud"
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/install-pico-cms/guide.fr-fr.md]:
```suggestion
excerpt: "Découvrez comment installer le CMS Pico sur un hébergement web OVHcloud"
```

File: pages/web_cloud/web_hosting/install-pico-cms/guide.fr-fr.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+updated: 2024-03-18
+---
+
+## Objectif
+
+Le **CMS** (**C**ontent **M**anagement **S**ystem) Pico vous permet de créer des sites web d'une manière simple et rapide. Conçu pour créer et gérer du contenu facilement grâce à des fichiers Markdown, Pico est idéal pour concevoir des sites web personnels, des portfolios ou des projets de petite entreprise.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/install-pico-cms/guide.fr-fr.md]:
```suggestion
Le **CMS** (**C**ontent **M**anagement **S**ystem) Pico permet de créer rapidement des sites web. Conçu pour créer et gérer du contenu facilement grâce à des fichiers Markdown, Pico est idéal pour concevoir des sites web personnels, des portfolios ou des projets de petites entreprises.
```

File: pages/web_cloud/web_hosting/install-pico-cms/guide.fr-fr.md [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+
+- Disposer d'une offre d'[hébergement web OVHcloud](https://www.ovhcloud.com/fr/web-hosting/).
+- Disposer d'un [nom de domaine](https://www.ovhcloud.com/fr/domains/).
+- Être connecté à votre [espace client OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/fr/&ovhSubsidiary=fr).
+
+## En pratique
+
+### Préparer l'installation
+
+Pour installer le CMS **Pico** sur votre offre d'[hébergement web](https://www.ovhcloud.com/fr/web-hosting/), quelques préparatifs sont nécessaires.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/install-pico-cms/guide.fr-fr.md]:
```suggestion
Pour installer le CMS **Pico** sur votre [hébergement web](https://www.ovhcloud.com/fr/web-hosting/), quelques préparatifs sont nécessaires.
```

File: pages/web_cloud/web_hosting/install-pico-cms/guide.fr-fr.md [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+
+- `content/` : contient les fichiers Markdown de votre contenu
+- `config/` : contient le fichier de configuration `config.yml` de Pico
+- `themes/` : contient les thèmes de votre site web
+- `assets/` : contient les ressources statiques comme des images, des feuilles de style CSS, des scripts JavaScript, etc.
+- `plugins/` : contient les plugins que vous souhaitez utiliser
+
+##### Configuration de base
+
+**1. Configurer votre site web** : ouvrez le fichier `config/config.yml` avec un éditeur de texte. C'est ici que vous pouvez configurer les paramètres de base de votre site web, tels que le titre, la description, l'URL ou les thèmes.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/install-pico-cms/guide.fr-fr.md]:
```suggestion
**1. Configurer votre site web** : ouvrez le fichier `config/config.yml` avec un éditeur de texte. Configurez ici les paramètres de base de votre site web, tels que le titre, la description, l'URL ou les thèmes.
```

File: pages/web_cloud/web_hosting/install-pico-cms/guide.fr-fr.md [hunk 8]
```diff
@@ -0,0 +1,113 @@
...
+
+**2. Changer le titre et la description de votre site web** : recherchez les lignes pour `site_title:` et `site_description:` dans le fichier `config/config.yml` pour mettre à jour le titre et la description de votre site web.
+
+**3. Changer le titre et la description de votre site web** : si vous avez un nom de domaine spécifique, recherchez la ligne pour `base_url:` dans le fichier `config/config.yml` pour mettre à jour le nom de domaine de votre site web, sinon, laissez la valeur `~` par défaut.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/install-pico-cms/guide.fr-fr.md]:
```suggestion
**3. Changer le titre et la description de votre site web** : si vous avez un nom de domaine spécifique, recherchez la ligne pour `base_url:` dans le fichier `config/config.yml` pour mettre à jour le nom de domaine de votre site web. Sinon, laissez la valeur `~` par défaut.
```

File: pages/web_cloud/web_hosting/install-pico-cms/guide.fr-fr.md [hunk 9]
```diff
@@ -0,0 +1,113 @@
...
+
+##### Ajouter du contenu
+
+**1. Créer des pages** : pour ajouter une nouvelle page à votre site web, créez un nouveau fichier Markdown (.md) dans le dossier `content/`. Le nom du fichier déterminera l'URL de la page. Par exemple, `about.md` sera accessible à http://votredomaine.com/about.
+
+**2. Écrire du contenu** : ouvrez le fichier Markdown avec un éditeur de texte et commencez à écrire votre contenu. Vous pouvez utiliser la syntaxe Markdown pour formater votre texte, créer des liens, insérer des images, etc. Par exemple, si vous voulez éditer la page principale (home) de votre site web, ouvrez le fichier `index.md` et insérez le contenu de votre choix.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/install-pico-cms/guide.fr-fr.md]:
```suggestion
**2. Écrire du contenu** : ouvrez le fichier Markdown avec un éditeur de texte et commencez à écrire votre contenu. Utilisez la syntaxe Markdown pour formater votre texte, créer des liens, insérer des images, etc. Par exemple, si vous voulez éditer la page principale (home) de votre site web, ouvrez le fichier `index.md` et insérez le contenu de votre choix.
```

File: pages/web_cloud/web_hosting/install-pico-cms/guide.fr-fr.md [hunk 10]
```diff
@@ -0,0 +1,113 @@
...
+
+**2. Modifier le thème** : pour changer de thème, mettez à jour la section `theme:` dans le fichier `config/config.yml` avec le nom du dossier du thème que vous souhaitez utiliser.
+
+**3. Personnaliser le thème** : vous pouvez modifier les fichiers du thème dans `themes/votretheme/` pour personnaliser l'apparence de votre site web. Cela peut inclure l'édition de fichiers HTML Twig, CSS, et JavaScript.
+
+##### Ajouter des plugins
+
+Pico permet d'étendre ses fonctionnalités avec des plugins.
+
+**1. Trouver des plugins** : consultez le [site web officiel de Pico](https://picocms.org/plugins/) pour consulter la liste des plugins disponibles.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/install-pico-cms/guide.fr-fr.md]:
```suggestion
**1. Trouver des plugins** : consultez le [site web officiel de Pico](https://picocms.org/plugins/) pour accéder à la liste des plugins disponibles.
```

File: pages/web_cloud/web_hosting/install-pico-cms/guide.fr-fr.md [hunk 11]
```diff
@@ -0,0 +1,113 @@
...
+
+[Tutoriel - Installer manuellement Drupal](/pages/web_cloud/web_hosting/cms_manual_installation_drupal)
+
+[Tutoriel - Installer manuellement PrestaShop](/pages/web_cloud/web_hosting/cms_manual_installation_prestashop)
+
+[Tutoriel - Installer manuellement un CMS sur mon hébergement](/pages/web_cloud/web_hosting/cms_manual_installation)
+
+Échangez avec notre communauté d'utilisateurs sur <https://community.ovh.com>.
+
+
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/install-pico-cms/guide.fr-fr.md]:
```suggestion
```

File: pages/web_cloud/web_hosting/install-pico-cms/guide.fr-fr.md [hunk 12]
```diff
@@ -0,0 +1,113 @@
...
+[Tutoriel - Installer manuellement Joomla!](/pages/web_cloud/web_hosting/cms_manual_installation_joomla)
+
+[Tutoriel - Installer manuellement Drupal](/pages/web_cloud/web_hosting/cms_manual_installation_drupal)
+
+[Tutoriel - Installer manuellement PrestaShop](/pages/web_cloud/web_hosting/cms_manual_installation_prestashop)
+
+[Tutoriel - Installer manuellement un CMS sur mon hébergement](/pages/web_cloud/web_hosting/cms_manual_installation)
+
+Échangez avec notre communauté d'utilisateurs sur <https://community.ovh.com>.
+
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/install-pico-cms/guide.fr-fr.md]:
```suggestion
```

File: pages/web_cloud/web_hosting/install-pico-cms/guide.fr-fr.md [hunk 13]
```diff
@@ -0,0 +1,113 @@
...
+[Tutoriel - Installer manuellement Joomla!](/pages/web_cloud/web_hosting/cms_manual_installation_joomla)
+
+[Tutoriel - Installer manuellement Drupal](/pages/web_cloud/web_hosting/cms_manual_installation_drupal)
+
+[Tutoriel - Installer manuellement PrestaShop](/pages/web_cloud/web_hosting/cms_manual_installation_prestashop)
+
+[Tutoriel - Installer manuellement un CMS sur mon hébergement](/pages/web_cloud/web_hosting/cms_manual_installation)
+
+Échangez avec notre communauté d'utilisateurs sur <https://community.ovh.com>.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/install-pico-cms/guide.fr-fr.md]:
```suggestion
Pour des prestations spécialisées (référencement, développement, etc), contactez les [partenaires OVHcloud](https://partner.ovhcloud.com/fr/directory/).

Échangez avec notre communauté d'utilisateurs sur <https://community.ovh.com>.
```

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @fabienOVH

**Pull request merged**